iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/string.inl

```diff
@@ -400,14 +326,16 @@ inline string<Capacity>& string<Capacity>::move(string<N>&& rhs) noexcept
 {
     static_assert(N <= Capacity,
                   "Assignment failed. The capacity of the given fixed string is larger than the capacity of this.");
-    uint64_t strSize = rhs.size();
+    const uint64_t strSize{rhs.size()};
     std::memcpy(&(m_rawstring[0]), rhs.c_str(), strSize);
     m_rawstring[strSize] = '\0';
     m_rawstringSize = strSize;
     rhs.clear();
     return *this;
 }
 
+// AXIVION DISABLE Style AutosarC++19_03-M5.17.1: This is not used as shift operator but as stream operator and does not require to implement '<<='
```

Comment:
Doesn't this warning only belong to the next construct? Why did you choose `AXIVION DISABLE Style` instead of `AXIVION Next Construct`?

Reply:
@FerdinandSpitzschnueffler `M5.17.1` violation has been reported multiple place in the file so decided to use the `DISABLE Style` instead of `AXIVION Next Construct` let me check and  may use the  `AXIVION Next Construct`